internal/db/sql.go

```diff
@@ -62,6 +62,40 @@ WHERE a.resource_id = $1
 ORDER BY a.id
 `
 
+const folderByIDSql = `
+SELECT id, name, order, user_id
+FROM public.folders
+WHERE id = $1
+`
+
+const foldersByUserIDSql = `
+SELECT f.id, f.name, f.order, f.user_id
+FROM public.folders f
+WHERE f.user_id = $1
+`
+
+const createFolder = `
+INSERT INTO public.folders (name, order, user_id, created_at, updated_at)
```

Comment:
While trying to test this locally, I discovered that using `order` as a column name without double quotes is a syntax error, since `order` is a SQL keyword (as in `order by`). This needs to be quoted as `"order"`.

Reply:
I fixed this along with a few other occurrences of this in other queries in https://github.com/ShelterTechSF/sheltertech-go/pull/115/commits/6b9dfea690bb046a4c376b06e1e7897d619add71